src/model/FormGenerator.js

```diff
@@ -29,7 +29,7 @@ export default class FormGenerator {
    *
    * @return Promise with generated form definition and form data
    */
-  static constructForm(structure, intl) {
+  static async constructForm(structure, intl) {
```

Comment:
I guess the comment of this method is not relevant anymore.

Reply:
I need to find the appropriate semantic but this is still appropriate, is is doing the same thing as before, just the dependency was bumped and so the implementation had to be changed, but the functionality remains the same